charts/agent-control/Chart.yaml

```diff
@@ -3,15 +3,15 @@ name: agent-control
 description: Bootstraps New Relic' Agent Control
 
 type: application
-version: 0.0.36-beta
+version: 0.0.37-beta
```

Comment:
remember the PR to update this in the guided install!

Reply:
If we follow the approach described here https://github.com/newrelic/helm-charts/pull/1565#discussion_r1913485335

I think we can skip this one, WDYT?

Reply:
fine for me!